### PR TITLE
Label drift fixes

### PR DESCRIPTION
**Pull request by nirhutnik**
Updated notebook and docstrings

### CODE REVIEW

File: docs/source/examples/checks/vision/distribution/train_test_label_drift.ipynb [hunk 1]
```diff
@@ -2,18 +2,84 @@
  "cells": [
```
Comment (noamzbr) [inline docs/source/examples/checks/vision/distribution/train_test_label_drift.ipynb:2]: <p>Think the examples here should be from vision</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/808/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (noamzbr) [inline docs/source/examples/checks/vision/distribution/train_test_label_drift.ipynb:2]: <p>can we make it show something interesting? Think Yuri has split this already</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/808/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (nirhutnik) [reply docs/source/examples/checks/vision/distribution/train_test_label_drift.ipynb:2]: <p>Agree, changed</p>

**Comment by nirhutnik** [comment]
<ol><li>Will be fixed in next iteration (the display in general)</li><li>Fixed</li></ol>

---

 <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/808/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Pull request merged**